lib/models.js

```diff
@@ -58,6 +58,16 @@ export class Path {
         return this.stat ? !this.stat.isDirectory() : null;
     }
 
+    isVisible() {
+      ignoredFiles = config.get('ignoredFiles');
```

Comment:
We want to declare variables with `const` if they don't change, or `let` if they do.